packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -129,28 +82,28 @@ exports.buildResolvableSet = ({
   defaultLocale,
 }) => {
   const resolvable = new Set()
-  existingNodes.forEach(n => {
-    if (n.contentful_id) {
-      if (process.env.EXPERIMENTAL_CONTENTFUL_SKIP_NORMALIZE_IDS) {
-        resolvable.add(n.contentful_id)
-      } else {
-        // We need to add only root level resolvable (assets and entries)
-        // derived nodes (markdown or JSON) will be recreated if needed.
-        // We also need to apply `fixId` as some objects will have ids
-        // prefixed with `c` and fixIds will recursively apply that
-        // and resolvable ids need to match that.
-        resolvable.add(`${fixId(n.contentful_id)}___${n.sys.type}`)
-      }
+  existingNodes.forEach(node => {
+    if (node.internal.owner === `gatsby-source-contentful`) {
+      // We need to add only root level resolvable (assets and entries)
+      // Derived nodes (markdown or JSON) will be recreated if needed.
+      const key = `${node.contentful_id}___${
+        node.sys.linkType || node.sys.type
```

Comment:
@axe312ger this was the missing fix to make the tests pass

Reply:
Hmm, I wonder how the nodes can be of type [Link](https://www.contentful.com/developers/docs/concepts/links/) while `entryList` should contain a fully resolved Contentful API response:

https://github.com/gatsbyjs/gatsby/blob/contentful_drop_fixids/packages/gatsby-source-contentful/src/gatsby-node.js#L169-L186